apps/dash-manufacture-spc-dashboard/assets/spc-custom-styles.css

```diff
@@ -0,0 +1,794 @@
+/* Table of contents
+––––––––––––––––––––––––––––––––––––––––––––––––––
+- Banner
+- Modal
+- Tabs
+- Main Dashboard Tab
+- Measurement Tab
+- Tables/Dropdown
+- Containers
+- Media Queries
+–––––––––––––––––––––––––––––––––––––––––––––––––– */
+
+* {
+  box-sizing: border-box;
+  -moz-box-sizing: border-box;
+}
```

Comment:
I realized you might still need this snippet to align things in mobile layout but perhaps have it as follows: 
```
#app-container * {
  box-sizing: border-box;
  -moz-box-sizing: border-box;
}
```

Reply:
fixed. 👍